src/Raven.Server/Smuggler/Documents/DatabaseDestination.cs

```diff
@@ -605,13 +637,20 @@ public void WriteDatabaseRecord(DatabaseRecord databaseRecord, SmugglerProgressB
                     progress.SortersUpdated = true;
                 }
 
-                if (currentDatabaseRecord?.ExternalReplications.Count == 0 &&
-                    databaseRecord?.ExternalReplications.Count > 0)
+                if (databaseRecord?.ExternalReplications.Count > 0)
                 {
                     if (_log.IsInfoEnabled)
                         _log.Info("Configuring external replications configuration from smuggler");
                     foreach (var replication in databaseRecord.ExternalReplications)
                     {
+                        currentDatabaseRecord?.ExternalReplications.ForEach(x =>
+                        {
+                            if (x.Name.Equals(replication.Name))
```

Comment:
Do we need to do OridinalIgnoreCase here?

Reply:
Yes